lib/index.ts

```diff
@@ -55,44 +54,42 @@ class BlueLinky extends EventEmitter {
 
   private onInit(): void {
     if (this.config.autoLogin) {
-      logger.info('Bluelinky is loging in automatically, to disable use autoLogin: false')
+      logger.info('Bluelinky is loging in automatically, to disable use autoLogin: false');
       this.login();
     }
   }
 
   public async login(): Promise<string> {
-      const response = await this.controller.login();
+    const response = await this.controller.login();
 
-      // get all cars from the controller
-      this.vehicles = await this.controller.getVehicles();
-      
-      logger.debug(`Found ${this.vehicles.length} on the account`);
+    // get all cars from the controller
+    this.vehicles = await this.controller.getVehicles();
 
-      this.emit('ready', this.vehicles);
-      return response;
+    logger.debug(`Found ${this.vehicles.length} on the account`);
+
+    this.emit('ready', this.vehicles);
+    return response;
   }
 
   async getVehicles(): Promise<Array<Vehicle>> {
-    return this.controller.getVehicles() || [];   
+    return this.controller.getVehicles() || [];
   }
 
-  public getVehicle(input: string): Vehicle|undefined {
+  public getVehicle(input: string): Vehicle | undefined {
     if (this.vehicles.length === 0) {
       throw new Error('No Vehicle found!');
     }
 
     try {
-      const foundCar = this.vehicles.find(car => {
-        car.vin === input ||
-        car.vehicleId === input
+      const foundCar = this.vehicles.find((car) => {
+        return car.vin === input || car.vehicleId === input;
       });
 
       if (!foundCar && this.vehicles.length > 0) {
-        return this.vehicles[0];
-      } 
-      
-      return foundCar;
+        throw new Error(`Could not find any vehicle based on the input`);
```

Comment:
I don't believe the user will under stand what "that input" menas. what about;
Could not afind vehicle with id: {input}